UefiTestingPkg/AuditTests/PagingAudit/README.md

```diff
@@ -1,5 +1,83 @@
 # Paging Audit
 
+## Contents
+
+- [Paging Audit](#paging-audit)
+  - [Contents](#contents)
+  - [DXE Paging Audit](#dxe-paging-audit)
+    - [DXE Driver](#dxe-driver)
+    - [DXE Shell App](#dxe-shell-app)
+      - [As a Shell-Based Unit Test](#as-a-shell-based-unit-test)
+      - [As a Paging Audit Collection Tool](#as-a-paging-audit-collection-tool)
+  - [SMM Paging Audit](#smm-paging-audit)
+    - [SMM Driver](#smm-driver)
+    - [SMM Shell App](#smm-shell-app)
+  - [Parsing the Paging Data](#parsing-the-paging-data)
+  - [Platform Configuration](#platform-configuration)
+    - [SMM Paging Audit Configuration](#smm-paging-audit-configuration)
+    - [DXE Paging Audit Driver Configuration](#dxe-paging-audit-driver-configuration)
+    - [DXE Paging Audit App Configuration](#dxe-paging-audit-app-configuration)
+  - [Copyright](#copyright)
+
+## DXE Paging Audit
+
+The DXE paging audit is a tool to analyze the page/translation tables of the platform to check
+compliance with Enhanced Memory Protection, debug paging related issues, and understand the

```

Comment:
Can you put a link to what is being referred to as "Enhanced Memory Protection"?

Reply:
Embedded a link to the Enhanced Memory Protection section of our Project Mu github.io page